<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deal</class>
 <widget class="QWidget" name="Deal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>订票系统</string>
  </property>
  <widget class="QFrame" name="HeaderBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>18</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>航班管理系统</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Avatar">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="btn_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>231</width>
     <height>931</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="home">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>首页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Deal_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>行程</string>
    </property>
   </widget>
   <widget class="QPushButton" name="favorite_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>收藏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Single_Center">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>个人中心</string>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>1031</width>
     <height>831</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_tickets">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <widget class="QTableWidget" name="tableWidget_tickets">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>1511</width>
       <height>661</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>出发地</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>目的地</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>出发时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>到达时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>价格（元）</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>可用座位</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>公司</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>操作</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>收藏</string>
      </property>
     </column>
    </widget>
    <widget class="QGroupBox" name="searchGroup">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>981</width>
       <height>71</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>搜索票务</string>
     </property>
     <widget class="QLabel" name="label_from">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>出发地：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_from">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>28</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="placeholderText">
       <string>例如：北京</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_to">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>目的地：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_to">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="placeholderText">
       <string>例如：上海</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_date">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>日期：</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>141</width>
        <height>33</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_type">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>30</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>类型：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_type">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>20</y>
        <width>131</width>
        <height>33</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <item>
       <property name="text">
        <string>全部</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>航班</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>火车</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>汽车</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="btn_search">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>20</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>770</y>
       <width>1011</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #7bed9f;    /* 浅草绿 */
    color: #333333;              /* 深色文字更清晰 */
    border: 1px solid #6bda8d;
    border-radius: 6px;
    padding: 10px 20px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_pageInfo">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>790</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>第        页/共</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>790</y>
       <width>21</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>790</y>
       <width>21</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_prev">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>790</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_next">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>790</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pageNum">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>790</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
